everviz/plugins/_crossplot_plugin/_dash_components/_crossplot_layout.py

```diff
@@ -0,0 +1,93 @@
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+def _create_crossplot_layout(graph_id):
+    return [
+        html.Div(
+            [
+                dcc.Graph(
+                    id=graph_id,
+                    style={
+                        "height": 900,
+                        "width": 900,
+                        "horizontal-align": "center",
+                        "display": "inline-block",
+                        "padding-left": "15%",
+                        "padding-right": "15%",
+                    },
+                    figure={"layout": {"font": {"size": 20}}},
+                )
+            ],
+            style={},
+        )
+    ]
+
+
+def _create_header(
+    radio_items_x_id, radio_items_y_id, dropdown_x_id, dropdown_y_id, dropdown_options
+):
+    return [
+        html.Div(
```

Comment:
The only difference on these two `Div`'s are the id's and the "X-axis" vs "Y-axis".

Reply:
The style is also a bit different for `Y-axis`